### PR TITLE
`bundle update` post install messages

### PR DESCRIPTION
**Pull request by smathy**
Currently post install messages are only displayed for `bundle install` not for `bundle update`.  This PR fixes that so they're displayed for both, and includes a small refactor of some duplicated functionality between `install` and `update`

### CODE REVIEW

File: spec/update/gems/post_install_spec.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  before do
+    gemfile <<-G
+      source "file://#{gem_repo1}"
+      gem 'rack', "< 1.0"
+      gem 'thin'
+    G
+
+    bundle "config #{config}" if config
+
+    bundle :install
```
Comment (segiddins) [inline spec/update/gems/post_install_spec.rb]: please use `bundle!`

**Comment by segiddins** [comment]
Thank you very much @smathy! 👍  from me!

File: spec/update/gems/post_install_spec.rb (hunk 1 above)
Comment (smathy) [inline spec/update/gems/post_install_spec.rb]: Oh thanks, I hadn't noticed that method.  I put it as a separate commit, not sure what the preference is here :)

**Comment by smathy** [comment]
Np @segiddins, really appreciate the time you spend to make bundler great.

**Comment by segiddins** [comment]
@bundlerbot r+

**Comment by smathy** [comment]
Thanks again.
